### PR TITLE
Change category to radio button

### PR DESCRIPTION
**Pull request by jpwalbran**

### CODE REVIEW

**Comment by jpwalbran** [comment]
Changes the category selection on the todos template to a radio button selection instead of the freeform text input.

File: client/todos.html [hunk 1]
```diff
@@ -9,22 +10,38 @@
   <script type="text/javascript" src="/javascript/util.js"></script>
   <script type="text/javascript" src="/javascript/todos.js"> </script>
 </head>
+
 <body>
-  <a href="/">Home</a> |
+  <a href="/index.html">Home</a> |
```
Comment (NicMcPhee) [inline client/todos.html:15]: Do we want to add an explicit `index.html` here? Isn't just using `/` more flexible?

File: client/javascript/todos.js [hunk 2]
```diff
@@ -7,8 +7,11 @@ function getFilteredTodos() {
   if(document.getElementById("owner").value != "") {
     url = url + "&owner=" + getFieldValue("owner");
   }
-  if(document.getElementById("category").value != "") {
-    url = url + "&category=" + getFieldValue("category");
+
+  let checkedCategory = document.querySelector('input[name="category"]:checked').value;
```
Comment (NicMcPhee) [inline client/javascript/todos.js:11]: Very nice!

File: client/users.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
   <script type="text/javascript" src="/javascript/users.js"></script>
 </head>
 <body>
-  <a href="/">Home</a> |
+  <a href="/index.html">Home</a> |
```
Comment (NicMcPhee) [inline client/users.html:13]: Same question about whether we need/want `index.html` here.

File: client/todos.html [hunk 4]
```diff
@@ -37,13 +54,15 @@
...
   <button type="button" id="getTodos">Get Todos</button>
 
-  <div style="margin-top: 20px;">Requested URL: <code id="requestUrl">(the requested API URL will appear here)</code></div>
+  <div style="margin-top: 20px;">Requested URL: <code id="requestUrl">(the requested API URL will appear here)</code>
+  </div>
   <pre id="jsonDump">JSON will go here</pre>
 
   <script>
-    window.onload = function() {
+    window.onload = function () {
```
Comment (NicMcPhee) [inline client/todos.html:62]: I didn't realize that the recommended formatting is to put a space here, but it seems to be. Nice catch!

**Review by NicMcPhee — COMMENTED** [review]
Nice improvement! My only open question is about whether we want to explicitly say `index.html` in the home links. Thoughts?

File: client/todos.html (hunk 1 above)
Comment (jpwalbran) [reply client/todos.html]: Probably not. It shouldn't really matter, but '/' is probably more flexible. This is now fixed with the most recent commit.

File: client/users.html (hunk 3 above)
Comment (jpwalbran) [reply client/users.html]: Same answer as above, fixed in the newest commit.

**Review by NicMcPhee — APPROVED** [review]

**Pull request merged**